app/controllers/api/v1/collections_controller.rb

```diff
@@ -22,5 +22,21 @@ def build_resource_for_create(create_params)
     add_user_as_linked_owner(create_params)
     super(create_params)
   end
-end
 
+  def add_relation(resource, relation, value)
```

Comment:
@marten, @parrish - I'm not 100% sold on the way i'm doing this as it implies we track the originating project link and not necessarily the project context the subject was collected in, e.g. subject uploaded to project A but reused in project B. 

It could instead default to linked project tracking if no project_id param is supplied? thus allowing the context to be passed in via link params.